src/panels/config/integrations/ha-config-entry-device-row.ts

```diff
@@ -204,6 +220,9 @@ class HaConfigEntryDeviceRow extends LitElement {
       ha-md-list-item {
         --md-list-item-leading-space: 56px;
       }
+      :host([narrow]) ha-md-list-item {
+        --md-list-item-leading-space: 36px;
```

Comment:
I like this one 👀  
```suggestion
        --md-list-item-leading-space: 16px;
```

Reply:
Thats no extra indent, just the default 😄 (fine by me...)